activesupport/lib/active_support/number_helper/number_to_currency_converter.rb

```diff
@@ -9,6 +9,8 @@ def convert
 
         if is_negative?(number)
```

Comment:
If we do that then following test fails - 

```
assert_equal("(0.00)", number_helper.number_to_currency(-0.0, {:unit => "", :negative_format => "(%n)"}))
```

As `negative_format` did not get assigned to `format`